geometry_msgs/Pose2D rearAxisPose
float64 velocity
float64 steeringAngle
bool autonomyEnabled